src/ui/widgets/WrapperWidgetUI.js

```diff
@@ -162,22 +162,23 @@ function WrapperWidgetUI(props) {
                 id='options-menu'
                 elevation={3}
                 anchorOrigin={{
-                  vertical: 'bottom',
+                  vertical: 'top',
                   horizontal: 'right',
                 }}
                 transformOrigin={{
                   vertical: 'top',
                   horizontal: 'right',
                 }}
                 anchorEl={anchorEl}
+                getContentAnchorEl={null}
                 keepMounted
                 open={open}
                 onClose={handleClose}
                 PaperProps={{
                   style: {
                     marginTop: '48px',
-                    maxHeight: '144px',
-                    width: '128px',
+                    maxHeight: '168px',
```

Comment:
Migrate it to `theme.spacing` using `useTheme` hook

Reply:
By defeault 1 spacing = 8, so 
```
marginTop:  theme.spacing(8),
maxHeight:  theme.spacing(21),
```